_summaries/bjorck2018understanding_1.md

```diff
@@ -1,19 +1,63 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{bjorck2018understanding}}
 # Please fill out info below
-author: # Your GitHub id
+author: # timchen0618
 score: # How did you like this paper 0(dislike) to 10(love)
 ---
 
 TODO: Summarize the paper:
-* What is the core idea?
-* How is it realized (technically)?
-* How well does the paper perform?
-* What interesting variants are explored?
+## Background
+Normalizing the input data to zero-mean and constant standard deviation has long been recognized as good practice in training neural networks. 
+Batch normalization (BN) extend this idea to the input of each layer and achieve immense success in various areas. 
+It enables faster training, larger learning rates, and higher testing accuracy (and better generalizability). 
+However, the reason for the improvement is to be debated. 
+(e.g. the paper that propose BN said it was solving "internal covariate shift", but some suggested otherwise)
+
+## Core idea 
+The paper pointed out that BN is beneficial to training neural networks mainly because it allow for larger learning rates. 
```

Comment:
I guess it will be good to mention that since network with BN and network without BN, if they have same learning rate, they can achieve same level of accuracy, which indicates that the success of BN is because it enables the use of higher learning rate.

Reply:
yeah! That would be more clear, thank you!